src/schema.json

```diff
@@ -50,6 +50,11 @@
           }
         },
         "additionalProperties": false
+      },
+      "outputPath": {
+        "type": "string",
+        "description": "Path to the ngsw.json in the production build output folder.",
```

Comment:
it's not so clear in the description that it's the path to the directory, not the file itself